R/version.R

```diff
@@ -240,6 +238,19 @@ format.version_sentinel <-
 .get_R_version <- function()
     getRversion()
 
+.get_sys_file <- function()
```

Comment:
Maybe this should implement the full test `nzchar(system.file(package = "BiocVersion"))` returning TRUE or FALSE